proxstar/tasks.py

```diff
@@ -151,7 +150,7 @@ def process_expiring_vms_task():
                             vm.name, vm.id
                         )
                     )
-                    send_stop_ssh_tunnel(vm.id)
+                    # send_stop_ssh_tunnel(vm.id) # TODO (willnilges): Remove target from targets file
```

Comment:
Personally not a fan of commented lines of code when we have git history. Not a blocker, just a comment